lte/gateway/c/core/oai/tasks/s1ap/s1ap_state_manager.cpp

```diff
@@ -111,17 +111,20 @@ void free_s1ap_state(s1ap_state_t* state_cache_p) {
       assoc_id = (sctp_assoc_id_t) keys->keys[i];
       ht_rc    = hashtable_ts_get(
           &state_cache_p->enbs, (hash_key_t) assoc_id, (void**) &enb);
-      AssertFatal(ht_rc == HASH_TABLE_OK, "eNB UE id not in assoc_id");
+      if (ht_rc != HASH_TABLE_OK) {
+        OAILOG_ERROR(LOG_S1AP, "eNB entry not found in eNB S1AP state");
+      }
       hashtable_uint64_ts_destroy(&enb->ue_id_coll);
```

Comment:
would not this lead to crash if `ht_rc != HASH_TABLE_OK`

Reply:
updated the if / else case